src/pages/my-claims/my-claims.tsx

```diff
@@ -14,12 +14,14 @@ import { isActive, useUserClaims } from '../../logic/claims';
 import { useUserPolicies, isActive as isPolicyActive } from '../../logic/policies';
 import styles from './my-claims.module.scss';
 
+type Filter = 'active' | 'inactive' | 'none' | null;
```

Comment:
Does it make sense to have `none` filter?

Reply:
Yeah, the `none` query param is for when both checkboxes are unchecked, and null (i.e. no query param) is for when both checkboxes are checked

Reply:
yeah, but if there is none you will not see any results... But I guess it's fine since it's the intuitive behaviour.